### ISSUE TITLE
basic media player functionallity

### ISSUE THREAD

**Issue by Commandcracker** [Commandcracker/YouCube#18]
add features like playlist support, shuffle, random, loop, volume, pause / play

**Comment by walksanatora** [comment]
what is the diffrence between shuffle and random

**Comment by walksanatora** [comment]
because now that i have a fork i think i can impl the shuffle

**Comment by walksanatora** [comment]
PR'd

**Comment by Commandcracker** [comment]
Shuffle will randomize the playlist order and random will start playback from a random point in the playlist.

**Comment by walksanatora** [comment]
oh wait i get it
so one completely changes the playlist
the other just skips some songs and puts them at the end

**Comment by Commandcracker** [comment]
Yes thats what i thought, but I did not think of the putting it back part ("and puts them at the end") :P

**Comment by walksanatora** [comment]
task shuffle was done by #87 so that can be marked as done
